markets/dagstore/lotusaccessor.go

```diff
@@ -58,6 +59,54 @@ func (m *lotusAccessor) Start(_ context.Context) error {
 	return m.readyMgr.FireReady(nil)
 }
 
+func (m *lotusAccessor) IsUnsealed(ctx context.Context, pieceCid cid.Cid) (bool, error) {
```

Comment:
The code duplication will be consolidated later, while we clean up the throttling primitives and we potentially collapse the `lotusAccessor` into the `LotusMount`.